catalog/spatial/geocoding/spatial-geocoding-geocoder/src/main/java/org/codice/ddf/spatial/geocoder/impl/GeoCoderServiceImpl.java

```diff
@@ -11,51 +11,40 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.spatial.geocoder.endpoint;
+package org.codice.ddf.spatial.geocoder.impl;
 
 import ddf.catalog.util.impl.ServiceSelector;
 import java.util.List;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.Response;
 import net.minidev.json.JSONArray;
 import net.minidev.json.JSONObject;
 import org.codice.ddf.spatial.geocoder.GeoCoder;
 import org.codice.ddf.spatial.geocoder.GeoResult;
+import org.codice.ddf.spatial.geocoding.GeoCoderService;
 import org.codice.ddf.spatial.geocoding.GeoEntryQueryException;
 import org.codice.ddf.spatial.geocoding.context.NearbyLocation;
 import org.opengis.geometry.DirectPosition;
 import org.opengis.geometry.primitive.Point;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-@Path("/")
-public class GeoCoderEndpoint {
+public class GeoCoderServiceImpl implements GeoCoderService {
 
   private ServiceSelector<GeoCoder> geoCoderFactory;
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(GeoCoderEndpoint.class);
-
-  public GeoCoderEndpoint(ServiceSelector<GeoCoder> geoCoderFactory) {
+  public GeoCoderServiceImpl(ServiceSelector<GeoCoder> geoCoderFactory) {
 
     if (geoCoderFactory == null) {
       throw new IllegalArgumentException(
-          "GeoCoderEndpoint(): constructor argument 'geoCoderFactory' may not be null.");
+          "GeoCoderServiceImpl(): constructor argument 'geoCoderFactory' may not be null.");
     }
 
     this.geoCoderFactory = geoCoderFactory;
   }
 
-  @GET
-  public Response getLocation(
-      @QueryParam("jsonp") String jsonp, @QueryParam("query") String query) {
+  public String getLocation(String jsonp, String query) {
```

Comment:
✏️ It would probably be good in this and all of these related PRs if the concrete REST classes that have been updated to implement interfaces had the `@Override` annotation added to the overridden methods.

Reply:
I'll add the `@Override`s to all the PRs